webcomponents/tool/src/main/frontend/js/grader/sakai-grader-rendering-mixin.js

```diff
@@ -709,12 +709,28 @@ export const graderRenderingMixin = Base => class extends Base {
             <div class="ms-2"><span class="grader-returned fa fa-eye" title="${this.i18n.returned_tooltip}" /></div>
           ` : ""}
         </div>
-        ${this.submission.groupId && this.submission.submittedTime ? html`<div class="grader-group-members">${this.submission.groupMembers}</div>` : ""}
+        ${this.submission.groupId && this.submission.submittedTime ? html`
+          <div class="grader-group-members">${this.submission.groupMembers}</div>
+        ` : ""}
         <div class="attachments">
+          ${this.submission.submittedText && this.submission.visible ? html`
+          <div>
+            <button type="button"
+                class="btn btn-transparent text-decoration-underline"
+                @click=${e => this.submittedTextMode = true}>
```

Comment:
Please fix the codacy issue: 
Expected parentheses around arrow function argument.

Thanks!

Reply:
I don't know why we have that rule in our codacy, tbh. We don't enforce that in the webcomponents project as I think it make's it more cluttered when you just have one argument.

Google agrees with me ...
https://google.github.io/styleguide/jsguide.html#formatting-grouping-parentheses

.. although I think the airbnb style guide forces them all the time. Also, we don't enforce parens in Java lambdas with single arguments.